cdap-app-fabric/src/main/java/io/cdap/cdap/app/store/Store.java

```diff
@@ -377,7 +377,7 @@ boolean scanApplications(ScanApplicationsRequest request, int txBatchSize,
   Collection<ApplicationSpecification> getAllAppVersions(ApplicationId id);
```

Comment:
This should be a versionless call (taking NamespaceID and application name)

Reply:
changed the method signature